index.php

```diff
@@ -49,36 +49,78 @@
         </nav>
     </div>
 </header>
-
+<?php $categories = ['Доски и лыжи', 'Крепления', 'Ботинки', 'Одежда', 'Инструменты', 'Разное']; 
```

Comment:
```suggestion
<?php 
$categories = ['Доски и лыжи',  'Крепления', 'Ботинки',  'Одежда',  'Инструменты',  'Разное']; 
```
Открытый тег php в данном случае должен быть без лишнего кода в строке. 
После запятой пробел.
Оформляй код правильно)